### PR TITLE
Native CLI packages

### PR DESCRIPTION
**Pull request by zachmullen**
This successfully builds the CLI as a single executable for me on MacOS aarch64. 

It seems that [cross-compilation is not supported](https://github.com/indygreg/PyOxidizer/issues/95).

### CODE REVIEW

File: .github/workflows/package.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+          pip install --upgrade pip
+          pip install tox
+      - name: Run packaging step
+        run: |
+          tox -e package
+      - name: Upload binary to release
+        run: |
+          gh release upload \
+            ${{ github.event.release.tag_name }} \
+            "$GITHUB_WORKSPACE/build/*/debug/install/isic*#${{ runner.os }} executable"
```
Comment (zachmullen) [inline .github/workflows/package.yml:28]: @danlamanna any idea how to test this without actually generating a release? I won't be convinced that this shell invocation works until we actually run the `gh release upload` command...

Comment (zachmullen) [reply .github/workflows/package.yml]: I'm just going to dump the files on an existing release as a test, I don't think it will do much harm.

**Comment by zachmullen** [comment]
This is ready for final review. I dumped the packages here for now for testing: https://github.com/ImageMarkup/isic-cli/releases/tag/v6.1.0

The zip approach has the virtue of not requiring users to `chmod +x`, and also allows the underlying command to retain the same name across platforms. (@brianhelba )

File: .github/zip_and_upload_package.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+if [[ "$runner_os" = "Windows" ]]; then
+    tar.exe -a -c -f $zipfile $executable
+else
+    zip $zipfile $executable
+fi
+
+gh release upload \
+    $tag_name \
+    "${zipfile}#${runner_os} executable" \
```
Comment (danlamanna) [inline .github/zip_and_upload_package.sh:27]: Is the `#` syntax or something? :thinking:

Comment (zachmullen) [reply .github/zip_and_upload_package.sh:27]: Yes. `gh release upload` splits the filename on the `#` if it's present, and whatever comes after is used as the display label in the list of files attached to the release.

**Comment by danlamanna** [comment]
LGTM! Merging and releasing :crossed_fingers:.

**Pull request merged**